src/wymeditor/iframe/default/wymiframe.html

```diff
@@ -21,5 +21,5 @@
 <title>WYMeditor iframe</title>
 <link rel="stylesheet" type="text/css" media="screen" href="wymiframe.css" />
 </head>
-<body class="wym_iframe"></body>
+<body></body>
```

Comment:
Does this mean that existing iframes that already contain `wym_iframe` won't work with keyboard shortcuts? That's a pretty big backwards incompatible change. We should remove it (if it exists), register shortcuts and then re-add it, if possible.

PolicyStat uses a custom `iframe` that would be affected by this, for example. At the very least, it should be called out as a backwards incompatible change so that folks know to update their iframes.